Sources/Brave/Frontend/Browser/Playlist/Browser/BrowserViewController+Playlist.swift

```diff
@@ -298,7 +298,8 @@ extension BrowserViewController: PlaylistScriptHandlerDelegate, PlaylistFolderSh
   func addToPlaylist(item: PlaylistInfo, folderUUID: String? = nil, completion: ((_ didAddItem: Bool) -> Void)? = nil) {
     PlaylistP3A.recordUsage()
     
-    if PlaylistManager.shared.isDiskSpaceEncumbered() {
+    if PlaylistManager.shared.isDiskSpaceEncumbered() && !Preferences.AppState.didShowStorageFullWarning.value {
```

Comment:
You probably don't need a preference :)
You can use a regular static boolean since it will reset once the app is terminated.

This way, if the app crashes, you don't have to worry about this preference resetting.